Hazelcast.Net/Hazelcast.IO.Serialization/SerializationService.cs

```diff
@@ -349,13 +348,13 @@ protected internal ISerializerAdapter SerializerFor(int typeId)
             if (typeId <= 0)
             {
                 var index = IndexForDefaultType(typeId);
-                if (index < ConstantSerializersSize)
-                {
+                if (index < ConstantSerializersSize && index != -SerializationConstants.DefaultTypeSerializable)
```

Comment:
missing curly bracket

Reply:
fixed

Reply:
index check would be better with:
`index < ConstantSerializersSize && index != IndexForDefaultType(SerializationConstants.DefaultTypeSerializable)`

Reply:
Changed